### ISSUE TITLE
No save file created on Linux (Flatpak) CloudCompare version 2.11.0

### ISSUE THREAD

**Issue by mz8i** [CloudCompare/CloudCompare#1460]
**Describe the bug**

I recently installed CloudCompare v2.11.0 from Flathub.

I did some processing on a point cloud from a .ply file, and wanted to save the result.
However, when I select the cloud and use the Save menu, no save file is created. Doesn't matter what file type, file name, or location I choose - there is no error, but no file appears.

When I use the batch export functionality from the Tools menu, the summary text file is saved, but again - the point cloud file isn't.

The fact that I will lose my work is not great, so I am wondering if there is any other way of pulling out the point cloud data from the open program, apart from saving it? Some place where I could copy the points as text to a different program? I haven't found one, but thought I'd ask...

**How to reproduce**

**Expected behaviour**

**Additional context**

I don't know if this is relevant, but the point cloud I am editing is from a aerial city scan in British National Grid coordinate system (EPSG:27700), so the global shift was suggested and applied by CC at the start. Not sure if the save file problem is related to this or a general problem with e.g. the Flatpak version.

**Your environment**
 - CC Version (or git hash/tag): 2.11.0
 - OS & Version: Manjaro Linux (kernel 5.10.38) with KDE Plasma 5.21.5
 - Graphics card: integrate Intel UHD 620
 - Qt Version (if compiling):

**Comment by tim-devereux** [comment]
So this issue is to do with flatpack permissions. You can set them easily by using Flatseal.

**Comment by chippey** [comment]
So, I'm having the same issue.  I checked Flatseal, and CC should have all the write permissions it needs, but I can't save out anything.

Your environment

    CC Version (or git hash/tag): 2.12 (flatpak)
    OS & Version: Pop! Linux 20.04 (kernel 5.15.15) with KDE Plasma 5.18.8
    Graphics card: nVidia Quadro RTX 4000
    Qt Version (if compiling):

**Comment by tim-devereux** [comment]
Yes, I am having this issue agian with the flatpack too. Maybe use snap (yeah i know) or the older official deb [package](https://ubuntu.pkgs.org/20.04/ubuntu-universe-arm64/cloudcompare_2.10.3-4build1_arm64.deb.html).

**Comment by h3ct0r** [comment]
I can confirm the SNAP installation works as expected

**Comment by chippey** [comment]
I can confirm this is still an issue with the latest flatpack install.  Still can't save or otherwise write any files.

**Comment by christian-rauch** [comment]
I can save files via a file manager Portal that opens once I press <kbd>Ctrl</kbd>+<kbd>S</kbd> or use the file menu to save a mesh. See https://github.com/flathub/org.cloudcompare.CloudCompare/issues/21#issuecomment-1211272250 for steps.

Can someone give the minimal steps to reproduce this and/or show a video of the issue?

**Comment by chippey** [comment]
Posting here too:
![Peek 2022-08-10 17-14](https://user-images.githubusercontent.com/3241883/184187730-bd18b876-5713-4af5-a38b-4afb4bbe189e.gif)

**Comment by philosadelphos** [comment]
I believe I've found the issue why ([snapshot]) didn't save the file to the specific folder.  I've CC 2.12.4 version in linux vera distro, which failed to save the «render to file» with output message:"[snapshot] Failed to save file", despite letting me save the project. The problem is with the extension (.png, .jpg, .tif, etc) that doesn't assume automaticaly in the input folder line after you have selected destination folder filename and extension. So, to not fail, we have to manually "force" the extension name in the input folder line after the path selection.

**Comment by christian-rauch** [comment]
> The problem is with the extension (.png, .jpg, .tif, etc) that doesn't assume automaticaly in the input folder line after you have selected destination folder filename and extension.

Is this an issue with CloudCompare or with Flatpak? Either way, I don't see how this can be fixed in this flatpak repo. Could you report the issue to the upstream projects and close this very issue? Or do you see a way how to resolve the issue here?

**Comment by philosadelphos** [comment]
> > The problem is with the extension (.png, .jpg, .tif, etc) that doesn't assume automaticaly in the input folder line after you have selected destination folder filename and extension.
> 
> Is this an issue with CloudCompare or with Flatpak? Either way, I don't see how this can be fixed in the [`org.cloudcompare.CloudCompare` flatpak repo](https://github.com/flathub/org.cloudcompare.CloudCompare).

It is with CC image export (if it is due to flatpak I don't really don't know).

**Comment by rjanvier** [comment]
Hi, beta version is available on Flathub if you want to test it
Please read [this](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) if you want to know the basic about beta channel on flathub. For CC, basically you have to do:

```
flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.cloudcompare.CloudCompare
```

You may need to make the beta version the default one with

```
flatpak make-current org.cloudcompare.CloudCompare beta
```
